metron-interface/metron-rest-client/src/main/java/org/apache/metron/rest/model/GrokValidation.java

```diff
@@ -0,0 +1,51 @@
+/**
```

Comment:
@merrimanr i think that's a really good idea. You're right, changing the spaces would add a lot of distractions.